stl/inc/xtree

```diff
@@ -1187,7 +1187,7 @@ public:
 
     _NODISCARD size_type max_size() const noexcept {
         return (_STD min)(
-            static_cast<size_type>((numeric_limits<difference_type>::max)()), _Alnode_traits::max_size(_Getal()));
+            static_cast<size_type>(_STD _Max_limit<difference_type>()), _Alnode_traits::max_size(_Getal()));

```

Comment:
If we drop `_STD` for `_Max_limit`, we'll unwrap most of the lines like this one

Reply:
I'm ok with this wrapping.